### DISCUSSION TITLE
CUSA01273 - Chasm

### DISCUSSION THREAD

**Discussion by Zehkarluz** [GoldHEN/GoldHEN_Cheat_Repository#14] [Cheat requests]
Inf health
Inf Magic
Inf Money
No hit from enemies, or invincibility
Inf itens on gain or sell, if possible
Thanks

**Comment by kmeps4** [comment]
Will check if there's a cheat available to add

**Comment by kmeps4** [comment]
https://github.com/GoldHEN/GoldHEN_Cheat_Repository/discussions/42#discussion-4222094
